.github/workflows/release_pipeline.yml

```diff
@@ -0,0 +1,74 @@
+name: Release Add-On Pipeline
+
+on: 
+  push:
+    branches: [ "pipeline-integration-test" ]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ${{matrix.os-type}}
+    outputs:
+      releaseVersion: ${{ steps.setVersion.outputs.version }}
+
+    strategy:
+      matrix:
+        os-type: [ windows-2019, macos-latest ]
+        version: [ 25, 26, 27 ]
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Checkout the submodule
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
```

Comment:
Is this necessary? We can use python3 command, although it wouldn't have the fixed 3.8 version.

Reply:
I prefer using fix versions, whenever possible for stability reasons. This also goes for the macos image version, we should decide on the exact version we want to use.

Reply:
I agree, using a fix version can save us some headache on the long run.